tox.ini

```diff
@@ -32,7 +34,7 @@ commands =
 [testenv:docs]
 passenv = *
 deps=
-    Sphinx==1.4.4
+    Sphinx
     sphinx_bootstrap_theme
```

Comment:
Tox is the single entry point to developing molecule.  If the user wants docs they run `$tox -e docs`.

They don't need to worry about installing requirements or anything.  Tox handles it for them.